firmware/controllers/algo/rusefi_enums.h

```diff
@@ -185,7 +185,7 @@ typedef enum {
 	MINIMAL_PINS = 99,
 	PROMETHEUS_DEFAULTS = 100,
 	SUBARUEJ20G_DEFAULTS = 101,
-
+    VAG_18_TURBO = 102,
```

Comment:
with a new enum value I would expect the firmware to not even compile since auto_generated was not refreshed.

let me add this value myself and you would need to rebase your version

Reply:
Great :-)

Thanks!